src/type_tracer/analyzer.rs

```diff
@@ -952,12 +976,19 @@ struct SymbolFiller<'a, THandler: TypeTraceHandler> {
 impl<'a, THandler: TypeTraceHandler> SymbolFiller<'a, THandler> {
   fn fill_module(&self, file_module: &mut EsmModuleSymbol, module: &Module) {
     let mut last_was_overload = false;
+    let mut last_override_key = 0;
```

Comment:
Fixes a bug where stuff like the following didn't work because it skipped the export statements:

```ts
function test(): void;
export { test };
```

Reply:
Can you add this one as a test?